### ISSUE TITLE
Feature request: colored tabs

### ISSUE THREAD

**Issue by 6r1d** [rxi/lite#204]
It will be really nice to see which tabs of Lite have unsaved changes by some color change: it will allow to remember more easily which files require saving.

**Comment by Tmpod** [comment]
This would be an easy change, but I don't know if rxi would like that idea. In the meanwhile, you can patch it yourself :)

[This block](https://github.com/rxi/lite/blob/master/data/core/rootview.lua#L339-L347) is where a tab is drawn and that `get_name` function is responsible for the asterisk (see [`DocView:get_name`](https://github.com/rxi/lite/blob/master/data/core/docview.lua#L86-L90)). In line 344, you can see the `draw_rect` call with `style.background` colour, and with a bit of simple logic you could make it use something other than that color.

Here's a quick mockup:
```lua
function Node:draw_tabs() 
  -- ...
  for i, view in ipairs(self.views) do
  -- ...
    if view == self.active_view then
      -- ...
    end
    if view.doc:is_dirty() then
      renderer.draw_rect(x, y, w, h, style.dirty_tab)
    end
    -- ...
  end
end
```

Then you have to set `style.dirty_tab = { common.color "#1337af" })` in your theme.

Hope this helps you :)

**Comment by 6r1d** [comment]
Thanks for the code!

> but I don't know if rxi would like that idea
Which is a very important question indeed, because current design is using asterisk in the end of a title, and it disappears quite easily.

**Comment by Tmpod** [comment]
Yep, should probably be changed to something like a little dot that persists on the tab even after the text starts to disappear.

**Comment by Gatsby1925** [comment]
I'm just facing this issue on closing the tabs when I'm done like there's no 'x' for closing it just keeps opening new ones it looks messy when you've opened a lot of tabs together.
